### ISSUE TITLE
Is it possible to access a wiki that's behind SSO?

### ISSUE THREAD

**Issue by hermannschwaerzlerUIBK** [Fannon/mobo#21]
Hi Simon,
our setup here at University of Innsbruck is a bit special: We are using our Singlle-Sign-On infrastructure (Shibolleth) to manage access to our wikis. That means we are using mod_shib on our Apache-servers and a not-logged in user gets a redirect response (HTTP status code 302) to our IDP where she has to login and gets redirected back to the originally requested page upon a successful login.

Can mobo be changed to handle this setting?
Now it fails in api.js with the error 
`Unhandled rejection Error: Request to API failed: HTTP status code was 302`

Unfortunately I am not a node-programmer so I can't do it myself. But maybe you can give me some hints? 
Or tell me it's too difficult? :-)

Greetings
Hermann

**Comment by Fannon** [comment]
Hello Hermann,

I've had similar issues with SSO but I did not find an easy way to use mobo in this envirionment, sorry! I guess it -could- be solved by injecting cookies. The upload library probably uses https://www.npmjs.com/package/request behind the scenes, but it would require some hacking to change the lower level request settings.

Another approach could be the use of a proxy, but I haven't tried this either!

Best,
Simon

**Comment by hermannschwaerzlerUIBK** [comment]
Hello Simon,
I was afraid of that so I came up with another solution: I created an instance that does not use SSO. I will use this as a staging instance and transfer the templates. forms asf to the production system later.

This works so far but I am encountering a new problem (I am unsure if I should open a new issue here at github):
The webserver of the wiki is configured to force all requests to https (all http-requests are redirected to https). My `mw_server_url` is set to start with "**https://wiki...**" but mobo stubbornly makes a POST-request to **http://wiki...** upon startup (and fails because it gets a status-code of 302 instead of 200) .
Am I missing something? Or do I have to disable that "https-forcing"?
